build/yamls/flow-aggregator/base/conf/flow-aggregator.conf

```diff
@@ -34,3 +34,6 @@
 # Provide format for records sent to the configured flow collector.
 # Supported formats are IPFIX and JSON.
 #recordFormat: "IPFIX"
+
+# Provide whether source and destination Pod labels will be included in the flow record.
+#includePodLabels: false
```

Comment:
I'd rather define a more generic configuration parameter that can be extended later:
```yaml
recordContents:
    podLabels: false
```

or `recordInformation` if it is preferred to `recordContents`.

This is the direction we are taking for other Antrea components as well. @srikartati what do you think?

Reply:
Yes. It is a good idea to make this extensible to multiple fields. However, this would only help with a small subset of fields that are added at the Flow Aggregator. We cannot truly disable the fields, especially the rest of Kubernetes metadata, which are added at the Flow Exporter--for this, we may need config map parameter in Antrea agent.  
The rest of the fields that are added at the Aggregator are source and destination statistics, which I feel is not so significant to have this toggle option. 

Maybe we could continue to have this option at the Aggregator and add this map-based data type in the Antrea Agent config map to disable at the Flow Exporter.

Reply:
I would say symmetry between the 2 configs is a good thing to have. There is also no telling whether we will include more information in the FlowAggregator in the future.

Reply:
It is a valid point that we may add new fields like pod labels in the future to the Flow Aggregator. Changing to map-based generic config parameter sounds good.

Reply:
@yanjunz97 could you address this ^

Reply:
Sure. Start to work on it

Reply:
> ```yaml
> recordContents:
>     podLabels: false
> ```

Should I use the name `podLabels` to replace all the `includePodLabels`? Or just use `podLabels` in the configmap

Reply:
done